Altis_Life.Altis/core/cop/fn_copMarkers.sqf

```diff
@@ -3,44 +3,34 @@
     Author: Bryan "Tonic" Boardwine
 
     Description:
-    Marks cops on the map for other cops. Only initializes when the actual map is open.
+    Marks cops on the map for other cops.
 */
-
 private _markers = [];
-private _cops = [];
-
-sleep 0.5;
-if (visibleMap) then {
-    _cops = playableUnits select {side _x isEqualTo west};
-
-    //Create markers
-    {
-        if !(_x isEqualTo player) then {
-            private _marker = createMarkerLocal [format ["%1_marker",_x],visiblePosition _x];
-            _marker setMarkerColorLocal "ColorBLUFOR";
-            _marker setMarkerTypeLocal "Mil_dot";
-            _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
 
-            _markers pushBack [_marker,_x];
-        };
-        true
-    } count _cops;
-
-    while {visibleMap} do {
-        {
-            _x params ["_marker","_unit"];
-            if (!isNil "_unit" && {!isNull _unit}) then {
-                _marker setMarkerPosLocal (visiblePosition _unit);
-            };
-            true
-        } count _markers;
-        if (!visibleMap) exitWith {};
-        sleep 0.02;
+{
+    if (side _x isEqualTo west) then {
+        private _marker = createMarkerLocal [format ["%1_marker",_x],getPosATL _x];
+        _marker setMarkerColorLocal "ColorBLUFOR";
+        _marker setMarkerTypeLocal "Mil_dot";
+        _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
+        _markers pushBack [_marker,_x];
     };
+} forEach (playableUnits - [player]);
```

Comment:
```suggestion
    private _marker = createMarkerLocal [format ["%1_marker", netId _x],getPosATL _x];
    _marker setMarkerColorLocal "ColorBLUFOR";
    _marker setMarkerTypeLocal "Mil_dot";
    _marker setMarkerTextLocal format ["%1", _x getVariable ["realname",name _x]];
    _markers pushBack [_marker,_x];
} forEach ((units west) - [player]);
```

makes use of new Syntax introduced in v2.02 - should be slightly faster. (also uses netId, see previous file)